lib/chef/resource/scm/_scm.rb

```diff
@@ -33,7 +33,8 @@
   default: "HEAD"
 
 property :user, [String, Integer],
-  description: "The system user that will own the checked-out code."
+  description: "The system user that will own the checked-out code.",
+  default_description: "`HOME` environment variable of the current user"
```

Comment:
Is this description accurate? If so, I'm not sure I understand it.

Reply:
It's ENV['HOME'] of the user running chef-client. I tried to write something brief so it fits in the format of the docs site.